File: Netbeans/Java/Lab6/src/model/NodAndNokFinder.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

/**
 *
 * @author prostrmk
 */
public class NodAndNokFinder {

    public static int finderOfNod(int a, int b) {
        return b == 0 ? a : finderOfNod(b,a % b);
    }

    public static int finderOfNok(int a, int b) {
        return (a/finderOfNod(a, b) * b);
    }

}

File: Csharp/Laba6/Laba6/Model/NodAndNokFinder.cs
using System;

namespace Laba6
{
	public class NodAndNokFinder
	{
		public static int FinderOfNod(int a, int b) {
			return b == 0 ? a : FinderOfNod(b,a % b);
		}
		public static int FinderOfNok(int a, int b) {
			return (a/FinderOfNod(a, b) * b);
		}
	}
}

